app/src/main/res/values/strings.xml

```diff
@@ -94,40 +97,40 @@
     <string name="recovery_code_input_hint_12">Word 12</string>
     <string name="recovery_code_error_empty_word">You forgot to enter this word.</string>
     <string name="recovery_code_error_invalid_word">Wrong word.</string>
-    <string name="recovery_code_error_checksum_word">Your code is invalid. Please check all words as well as their position and try again!</string>
+    <string name="recovery_code_error_checksum_word">Your code is invalid. Please check all the words as well as their position and try again.</string>
     <string name="recovery_code_verification_ok_title">Recovery code verified</string>
-    <string name="recovery_code_verification_ok_message">Your code is correct and will work for restoring your backup.</string>
+    <string name="recovery_code_verification_ok_message">Your code is correct, so it can restore your backup.</string>
     <string name="recovery_code_verification_error_title">Incorrect recovery code</string>
     <string name="recovery_code_verification_error_message">You have entered an invalid recovery code. Please try again!\n\nIf you have lost your code, tap on \'Generate new code\' below.</string>
     <string name="recovery_code_verification_try_again">Try again</string>
     <string name="recovery_code_verification_generate_new">Generate new code</string>
     <string name="recovery_code_verification_new_dialog_title">Wait one second…</string>
-    <string name="recovery_code_verification_new_dialog_message">Generating a new code will make your existing backups inaccessible. We\'ll try to delete them if possible.\n\nAre you sure you want to do this?</string>
-    <string name="recovery_code_recreated">New recovery code has been created successfully</string>
+    <string name="recovery_code_verification_new_dialog_message">Do you want to generating a new code, making your existing backups inaccessible? They will be deleted if possible.</string>
+    <string name="recovery_code_recreated">New recovery code created</string>
     <string name="recovery_code_auth_title">Re-enter your screen lock</string>
```

Comment:
What does this mean?

Reply:
Copied from AOSP Settings, I think. re-enter whatever you use on the screen lock / lock screen.